aws/README.md

```diff
@@ -8,12 +8,14 @@ Install `eksctl` and `kubectl` to interact with Amazon EKS and the Kubernetes co
 
 You can learn more about `eksctl` [here](https://eksctl.io/).
 
-To configure kubectl to talk to the healthmapidha cluster, do:
+To configure kubectl to talk to the ghdsi cluster, do:
 
 ```shell
-aws eks --region us-east-1 update-kubeconfig --name healthmapidha
+aws eks --region us-east-1 update-kubeconfig --name ghdsi
```

Comment:
I can't connect via kubectl. Did you update the `aws-auth` ConfigMap? If so, could be a problem with my local authenticator, but checking to see if this was done first.

Reply:
Solved, and added documentation on how to get access in the readme.